deps/cuda1.jl

```diff
@@ -3,6 +3,244 @@
 fp = open("cuda1.cu","w")
 using Knet: unary_ops
 
+
+# Digamma
```

Comment:
better put all these functions at the end of the file

Reply:
I can do this. Or is there a way to put all the codes in a single file?